src/cli/repo/repo.go

```diff
@@ -84,3 +103,65 @@ func connectCmd() *cobra.Command {
 func handleConnectCmd(cmd *cobra.Command, args []string) error {
 	return cmd.Help()
 }
+
+func maintenanceCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   maintenanceCommand,
+		Short: "Run maintenance on an existing repository",
+		Long:  `Run maintenance on an existing repository to keep it running smoothly`,
```

Comment:
Not sure I have a good suggestion for replacement, but `to keep it running smoothly` doesn't strike me as useful; it's sort of a generic extension to the notion of "maintenance".  Also, Corso always runs smoothly.  Not a bump or pothole in sight, amirite? 😉 

Spitballing: `to keep memory usage low`?  `to keep processing time down`?  `to compress backup history`?

Reply:
Run maintenance on an existing repository to optimize performance and storage use.